src/components/search/Search.jsx

```diff
@@ -113,6 +114,7 @@ const Search = () => {
 
   return (
     <>
+      <ExpiredSubscriptionModal />
```

Comment:
I see this question from the [related ticket](https://2u-internal.atlassian.net/browse/ENT-9617):

```
[Maham] Should this be shown on just 2 sections as mentioned or all pages of learner portal?
```

[curious] Is this still an open question? Or was there a decision made to only show it on the explicitly mentioned routes? If it is intended to be on _all_ pages, it might be able to live in `EnterprisePage` (food for thought).

Reply:
@adamstankiewicz The decision was made to show it only on explicitly mentioned routes - dashboard, search, course about.

Reply:
Thanks for sharing the decision/update! 😄